gallery/src/CodeExample.tsx

```diff
@@ -28,12 +28,20 @@ const removeLicense = (content: string) =>
 
 const CodeExample: FunctionComponent<Props> =
   function CodeExample({ content, language }): ReactElement<Props> {
-    const licenseStrippedContent = removeLicense(content);
+    const licenseStrippedContent = removeLicense(content),
+        headerLanguagePart = language ? ` (${language.toUpperCase()})` : '';
 
     return (
-      <SyntaxHighlighter language={language || 'tsx'} style={atomDark}>
-        {licenseStrippedContent}
-      </SyntaxHighlighter>
+      <section className="gallery-example">
```

Comment:
I think this class should be `.gallery-example-code`.

Reply:
> I think this class should be .gallery-example-code.

What would be the purpose of this class?

Reply:
> > I think this class should be .gallery-example-code.
> 
> What would be the purpose of this class?

I'm just trying to think ahead and anticipate future design changes. If we put these CSS classes in now they give us more flexibility in the future. Not having them might mean that we require the assistance of a JS dev to make a comparatively simple design change.